native/src/main/java/io/ballerina/stdlib/file/nativeimpl/Utils.java

```diff
@@ -50,16 +50,6 @@
 
 import static java.nio.file.FileVisitResult.CONTINUE;
 import static java.nio.file.FileVisitResult.SKIP_SUBTREE;
-import static org.ballerinalang.stdlib.file.utils.FileConstants.COPY_ATTRIBUTES;
-import static org.ballerinalang.stdlib.file.utils.FileConstants.EXISTS;
-import static org.ballerinalang.stdlib.file.utils.FileConstants.IS_DIR;
-import static org.ballerinalang.stdlib.file.utils.FileConstants.IS_SYMLINK;
-import static org.ballerinalang.stdlib.file.utils.FileConstants.METADATA;
-import static org.ballerinalang.stdlib.file.utils.FileConstants.NO_FOLLOW_LINKS;
-import static org.ballerinalang.stdlib.file.utils.FileConstants.READABLE;
-import static org.ballerinalang.stdlib.file.utils.FileConstants.RECURSIVE;
-import static org.ballerinalang.stdlib.file.utils.FileConstants.REPLACE_EXISTING;
-import static org.ballerinalang.stdlib.file.utils.FileConstants.WRITABLE;
```

Comment:
Is this intentional?

Reply:
Those changed automatically when the directory name was refactored.

Reply:
That's because of the default settings of the IDE. You can always change them if you want.